src/api/spec/features/webui/projects_spec.rb

```diff
@@ -15,20 +15,42 @@
     expect(page).to have_css('h3', text: project.title)
   end
 
-  it 'changing project title and description' do
-    login user
-    visit project_show_path(project: project)
+  describe 'changing project title and description' do
+    context 'when accepting the changes' do
+      it 'updates the project title, description and url' do
```

Comment:
Is it really updating the project's URL? It doesn't seem to be the case.

Reply:
Fixed